packages/owl-bot/package.json

```diff
@@ -47,11 +47,13 @@
     "mocha": "^8.2.1",
     "nock": "^13.0.6",
     "sinon": "^9.2.3",
-    "typescript": "~4.1.3"
+    "typescript": "~4.1.3",
+    "uuid": "^8.3.2"
   },
   "dependencies": {
     "@google-cloud/cloudbuild": "^2.0.6",
     "@octokit/rest": "^18.1.0",
+    "firebase-admin": "^9.4.2",
```

Comment:
In most cases, we would want to use `@google-cloud/firestore` here ... but I will read on!

Reply:
I followed the instructions here:
https://firebase.google.com/docs/firestore/quickstart